src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceFactory.char.cs

```diff
@@ -16,25 +16,15 @@ public abstract class ReadOnlySequenceFactoryChar
 
         internal class StringTestSequenceFactory : ReadOnlySequenceFactory<char>
         {
-            static string s_stringData = InitalizeStringData();
```

Comment:
Why remove the static that does the string data generation only once rather than for each call to CreateOfSize?

Reply:
Because it generates static length that was not enough. hat correct length is enough?